CIScripts/Test/Tests/vRouterAgentService.Tests.ps1

```diff
@@ -40,7 +44,9 @@ Describe "vRouter Agent service" {
 
     Context "given vRouter Forwarding Extension is NOT running" {
         It "crashes" {
-            Assert-AgentProcessCrashed -Session $Session -After $TimeBeforeCrash
+            {{ Test-HasAgentProcessCrashed -Session $Session -After $BeforeCrash } `
+                | Retry-UntilTrue -IntervalInSecs 1 -TimeoutInSecs 60 } `
+                | Should Not Throw
```

Comment:
Although I prefer not using `Should Not Throw` here at all, it seems I'm in a minority.

Anyway, I my previous comments about nested blocks applies, please use more obvious indentation or, preferably, pass the script block as explicit parameter to `Should`.